flocker/restapi/docs/test/test_publicapi.py

```diff
@@ -69,7 +69,7 @@ def test_must_be_documented(self):
         app = Klein()
 
         @app.route(b"/", methods=[b"GET"])
-        def f():
+        def dummy_f():
```

Comment:
Yeah, it's in the default config alongside "_" as an unused variable, I
didn't want to define the a function called "_", so used dummy. Open
to changing that.